modules/rosa-additional-principals-create.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-accessing-cluster.adoc
+// * rosa_install_access_delete_clusters/rosa-sts-accessing-cluster.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-additional-principals-create_{context}"]
+= Adding additional principals while creating your ROSA cluster
+
+Use the `--additional-allowed-principals` argument to permit access through other accounts and roles.
+
+.Procedure
+
+. In your terminal, add `--additional-allowed-principals` to the `rosa create cluster` command, similar to the following:
```

Comment:
suggest rephrasing as:

Use --additional-allowed-principals field in the 'rosa create cluster' CLI command, as shown below: